scAR/test/test_scAR.ipynb

```diff
@@ -0,0 +1,141 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ac5906d1-7f3a-4661-8235-8bca20ec9c45",
+   "metadata": {},
+   "source": [
+    "## integration test"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "aa6514e7-19b3-4b91-9f6f-e2ad125bf9d7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from scAR import model\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9f1b9ec0-8452-4f17-983a-324e2d2e1920",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "raw_count = pd.read_pickle('/da/onc/bfx/research/shengca1/github/scAR/scAR/test/raw_counts.pickle')\n",
```

Comment:
Can you please remove these absolute paths, because they expose internal paths?

Reply:
Thanks for reminding me. It's done.